src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt

```diff
@@ -883,6 +886,25 @@ class AdminController(
     return organization(organizationId)
   }
 
+  @PostMapping("/exportReport")
+  fun exportReport(
+      @RequestParam organizationId: OrganizationId,
+      @RequestParam reportId: ReportId,
+      redirectAttributes: RedirectAttributes
+  ): String {
+    requirePermissions { createReport(organizationId) }
+
+    try {
+      reportService.exportToGoogleDrive(reportId)
```

Comment:
Overwriting was pretty handy when I was testing this locally. It would have been possible to try to detect unchanged files (write a checksum or something) but like you said, it's an admin action so I didn't think it was worth optimizing the uploads.